packages/server/core/src/base/adapters/node/middlewares/serverManifest.ts

```diff
@@ -26,10 +26,15 @@ async function getServerManifest(
       );
 
       const renderBundlePath = path.join(pwd, route.bundle || '');
-
+      const dynamicImport = (filePath: string) => {
```

Comment:
@2heal1  some modern.js tests failed due to this PR, can you take a look?

<img width="804" alt="image" src="https://github.com/web-infra-dev/modern.js/assets/22373761/734f570c-e652-4b19-a52b-684e4ff155c0">

https://github.com/web-infra-dev/modern.js/actions/runs/8777467016/job/24082441052

Reply:
https://github.com/web-infra-dev/modern.js/pull/5672 this pr will fix the issue